### PR TITLE
chore: fix CVE-2021-23343

### PR DESCRIPTION
**Pull request by dleard**

### CODE REVIEW

File: app/yarn.lock [hunk 1]
```diff
@@ -10201,7 +10201,7 @@ path-key@^3.0.0, path-key@^3.1.0:
   resolved "https://registry.yarnpkg.com/path-key/-/path-key-3.1.1.tgz#581f6ade658cbba65a0d3380de7753295054f375"
   integrity sha512-ojmeN0qd+y0jszEtoY48r0Peq5dwMEkIlCOu6Q5f41lfkswXuKtYrhgoTpLnyIcHm24Uhqx+5Tqm2InSwLhE6Q==
 
-path-parse@^1.0.6:
+path-parse@^1.0.7:
   version "1.0.6"
```
Comment (pbastia) [inline app/yarn.lock:10205]: I think you meant to change the second "1.0.6" instead

File: app/yarn.lock [hunk 2]
```diff
@@ -10202,7 +10202,7 @@ path-key@^3.0.0, path-key@^3.1.0:
   integrity sha512-ojmeN0qd+y0jszEtoY48r0Peq5dwMEkIlCOu6Q5f41lfkswXuKtYrhgoTpLnyIcHm24Uhqx+5Tqm2InSwLhE6Q==
 
 path-parse@^1.0.6:
-  version "1.0.6"
+  version "1.0.7"
   resolved "https://registry.yarnpkg.com/path-parse/-/path-parse-1.0.6.tgz#d62dbb5679405d72c4737ec58600e9ddcf06d24c"
   integrity sha512-GSmOT2EbHrINBf9SR7CDELwlJ8AENk3Qn7OikK4nFYAu3Ote2+JYNVvkpAEQm3/TLNEJFD/xZJjzyxg3KBWOzw==
```
Comment (pbastia) [inline app/yarn.lock:10207]: Just need to remove those 2 lines and re-run yarn install

**Review by pbastia — CHANGES_REQUESTED** [review]
just a tiny yarn comment, looks great otherwise!

**Comment by dleard** [comment]
closes #1955

**Review by pbastia — APPROVED** [review]
:shipit: 👌

**Pull request merged**